apps/dialtone-documentation/docs/design/size/index.md

```diff
@@ -0,0 +1,155 @@
+<div>
+
+The size of some components is determined by their content. Most expand vertically, while a select few expand horizontally.
+
+</div>
+<dt-stack direction="column" gap="500">
+<iframe style="border: 0px solid rgba(0, 0, 0, 0.1); border-radius: 8px" width="100%" height="320" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FQe6cz41vPBozP4PhgGqFin/Docs-Protos?page-id=0%3A1&type=design&node-id=5-25749&viewport=-1055%2C122%2C0.69&t=iCwsUiY6dBufHiTQ-8&scaling=min-zoom&starting-point-node-id=5%3A25739&hotspot-hints=0&hide-ui=1"></iframe>
+<p class="d-body-base d-fc-tertiary">This tooltip has a `min-width`, and its height depends on its content.</p>
+</dt-stack>
+<dt-stack direction="column" gap="500">
+<iframe style="border: 0px solid rgba(0, 0, 0, 0.1); border-radius: 8px" width="100%" height="320" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FQe6cz41vPBozP4PhgGqFin/Docs-Protos?page-id=0%3A1&type=design&node-id=5-25744&viewport=-1055%2C122%2C0.69&t=iCwsUiY6dBufHiTQ-8&scaling=min-zoom&starting-point-node-id=5%3A25744&hotspot-hints=0&hide-ui=1"></iframe>
+<p class="d-body-base d-fc-tertiary">This button has a fixed `height`, a `min-width` and grows horizontally depending on its content.</p>
+</dt-stack>
+</div>
+
+### Responsiveness
+
+<div class="d-d-grid d-gg24 d-g-cols3 md:d-g-cols1">
+<div>
+
+Adapting to different devices, sizes should be fluid across breakpoints. While most Dialtone components are responsive, their fluidity depends on the layout they live within.
+
+</div>
+<dt-stack class="d-gc2" direction="column" gap="500">
+<div>
+<iframe style="border: 0px solid rgba(0, 0, 0, 0.1); border-radius: 8px" width="100%" height="320" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FQe6cz41vPBozP4PhgGqFin/Docs-Protos?page-id=0%3A1&type=design&node-id=5-25679&viewport=-722%2C-310%2C0.54&t=rVgNK5NwIXaPxkHq-8&scaling=min-zoom&starting-point-node-id=5%3A25663&hotspot-hints=0&hide-ui=1"></iframe>
+</div>
+</dt-stack>
+</div>
+
+### Accessibility
+
+<div class="d-d-grid d-gg24 d-g-cols1 md:d-g-cols1">
+
+To ensure clickable and interactive areas are easily accessible, we recommend a minimum area of `var(--dt-size-300)`, except for links within text. In Dialtone we've applied these to components such as buttons, toggles, checkboxes, radio buttons, dropdowns, and inputs.
+
+</div>
+
+## Tokens
+
+Here are some frequently used tokens. For a complete list, visit the [Size Tokens](/tokens/sizing/) section.
+
+<token-table category="size" :tokenList="tokenList" />
+
+WIP: Tokens decriptions/usage in progress.
+
+**Note:** By Feb 2024, we aim to integrate **sizing** units into Figma Variables. This will simplify the process of setting these tokens on width, min-width, height, min-height in your Figma files, making it easier for engineers to obtain the correct variable for each case.
```

Comment:
For both Size and Space, use the Info variant of [Notice](https://dialtone.dialpad.com/components/notice.html#base-styles) for this line.

Reply:
(Just the "Note:" part, now the above "WIP" part.

Reply:
Great! Thank youuu